.babelrc

```
@@ -1,27 +1,10 @@
 {
   "plugins": [
     "@babel/plugin-syntax-dynamic-import",
-    "@babel/plugin-transform-runtime",
-    "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-transform-template-literals",
-    "@babel/plugin-proposal-object-rest-spread",
-    "@babel/plugin-transform-destructuring"
+    "@babel/plugin-transform-runtime"
   ],
   "presets": [
-    [
-      "@babel/preset-env",
-      {
-        "useBuiltIns": "entry",
-        "corejs": "2",
-        "targets": {
-          "node": "current",
-          "ie": "11",
-          "firefox": "67",
-          "chrome": "75",
-          "edge": "44"
-        }
-      }
-    ],
+    "@babel/preset-env",
```

Why removing present-env config? I think we could target core-js 3 now

---

`targets` can be moved to the top level (I left a comment) but we still need useBuiltIns and corejs:
```suggestion
    [
      "@babel/preset-env",
      {
        "useBuiltIns": "entry",
        "corejs": "3.25"
      }
    ],
```

---

Well, we can discuss the `preset-env` config, but with these changes the idea is to let `preset-env` handle things with core-js instead of having it installed. 

As it is right now, `preset-env` is following browserslist default configuration, which is: `> 0.5%, last 2 versions, Firefox ESR, not dead`. More information about this can be found [here](https://github.com/browserslist/browserslist#queries). Maybe we could discuss which browsers we want to target.